src/agent/onefuzz-agent/src/tasks/report/crash_report.rs

```diff
@@ -139,9 +149,57 @@ impl CrashTestResult {
         }
         Ok(())
     }
+
+    pub async fn save_regression(
```

Comment:
Could we add doc comments to all of these new public methods and types? It would help to know the intention at a glance, including where we expect things to get saved.

Reply:
(and what's supposed to happen for the repro/no repro)